2018-10-20 20:27:33 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Logger.Supervisor'}
     Context:    child_terminated
     Reason:     normal
     Offender:   [{pid,<0.499.0>},{id,'Elixir.Logger.ErrorHandler'},{mfargs,{'Elixir.Logger.Watcher',start_link,[{error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}]}},{restart_type,permanent},{shutdown,5000},{child_type,worker}]

2018-10-20 20:27:34 =ERROR REPORT====
Error in process <0.613.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:27:35 =ERROR REPORT====
Error in process <0.629.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:27:36 =ERROR REPORT====
Error in process <0.645.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:27:37 =ERROR REPORT====
Error in process <0.661.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
